homeassistant/components/zwave_js/discovery.py

```diff
@@ -745,9 +752,15 @@ def async_discover_values(node: ZwaveNode) -> Generator[ZwaveDiscoveryInfo, None
 
             # resolve helper data from template
             resolved_data = None
-            additional_value_ids_to_watch = None
+            additional_value_ids_to_watch = set()
             if schema.data_template:
-                resolved_data = schema.data_template.resolve_data(value)
+                try:
+                    resolved_data = schema.data_template.resolve_data(value)
+                except UnknownValueData as err:
+                    LOGGER.error(
+                        f"Discovery for value {value} will be skipped: {err.args[0]}"
```

Comment:
Do we need to pass the first exception argument explicitly or can we just pass the exception to the log?